### PR TITLE
Encoded txt files as UTF-8

### PR DESCRIPTION
**Pull request by seanknox**

### CODE REVIEW

**Comment by jasonlally** [comment]
Thanks for doing that.  Did you do this manually or programmatically?  Reason I ask, is that our vendor will be pushing updates to FTP that we then automatically add and commit to the repo.  I'm seeing if they can just make sure to save with UTF-8 encoding on their end, but just in case, I may need to script this so it doesn't have to be done manually.

**Comment by seanknox** [comment]
Programmatically. Here's my quick hack:

``` ruby
require 'charlock_holmes'

detector = CharlockHolmes::EncodingDetector.new

ARGV.each do|f|
  content = File.read(f)
  detection = detector.detect(content)
  puts "#{f} encoding: #{detection[:encoding]}"
  utf8_encoded_content = CharlockHolmes::Converter.convert content, detection[:encoding], 'UTF-8'
  File.write(f, utf8_encoded_content)
end
```

File: Port.txt [hunk 1]
```diff
@@ -243,24 +243,24 @@ SEC. 5.8.  EXCEPTIONS.
...
-	(F)	Exhibition�Public display of manufactured goods, plants or other objects but not displays by street artists licensed pursuant to Part II, Chapter VIII (Police Code) of the San Francisco Municipal Code.
-	(G)	Leafletting�Distributing leaflets, handbills, notices or any written material to the public.
-	(H)	Orchestra�A group of musicians playing string, percussion, brass and wood wind instruments.
-	(I)	Parade�Any march or other organized movement or person from place to place or about a place.
-	(J)	Petitioning�Requesting persons to sign a petition.
-	(K)	Publicize�To inform the public of a planned event by means of newspaper articles or notices, radio or television stories or notices, announcements in public places, leafletting, posting signs or written notices in places viewed by the public, or by other means calculated to notify the public of an event.
-	(L)	Soliciting�Requesting persons to contribute money or anything else of value for charitable, religious or political cause.
+	(A)	Amusement Park RidesRides of the type normally found in amusement parks or carnivals, such as ferris wheels.
```
Comment (jasonlally) [inline Port.txt:252]: #4 In the original Municiap Code, these are em dashes, I believe, encoding them removes them. Thoughts on why this is happening?

**Comment by seanknox** [comment]
That's possible. I'm not sure there's a way to have the transcoder be a bit smarter about characters like that, but I'll look. Think the best way forward is to have the vendor encode as UTF-8 directly.

**Comment by jasonlally** [comment]
#2 addressed

**Pull request merged**